lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1406,6 +1406,17 @@ func (w *WalletKit) fundPsbtInternalWallet(account string,
 				return err
 			}
 
+			for _, utxo := range utxos {
+				// Don't use inputs used by the sweeper.
+				if utxo.Confirmations < 1 &&
```

Comment:
Are negative confirmations a thing? If not, can we do `== 0` instead for clarity?

Reply:
Yes they are filtered by `func (w *Wallet) ListUnspent` for the -1 case so we are good to use 0 here.